config/client.go

```diff
@@ -252,6 +252,7 @@ func GenerateLimitedClientConfig(c *model.Config, diagnosticId string, license *
 	props["HasImageProxy"] = strconv.FormatBool(*c.ImageProxySettings.Enable)
 
 	props["PluginsEnabled"] = strconv.FormatBool(*c.PluginSettings.Enable)
+	props["EnableMarketplace"] = strconv.FormatBool(*c.PluginSettings.EnableMarketplace)
```

Comment:
Should this be under `GenerateClientConfig` and not `GenerateLimitedClientConfig`? as we don't really need to expose this to untrusted clients.